app/_data/docs_nav_ee_1.3-x.yml

```
@@ -283,7 +283,7 @@
     - text: Using the Editor
       url: /developer-portal/using-the-editor
     - text: Networking
-      url: /developer-portal/networking
+      url: /property-reference/#developer-portal-section
```

```suggestion
      url: /property-reference/#dev-portal
```
Looks like it used to be just dev-portal for two releases.